### ISSUE TITLE
Parent triple map support

### ISSUE THREAD

**Issue by nicky508** [zazuko/xrm#31]
To make nested structures, parenttriple maps need to be added to the DSL, example:

```
ex:Object rdf:type class:Object ;
prop:lotSize [ 	schema:value 11050.07095540000;
			schema:unitCode 'FTK' ];
```

**Comment by mchlrch** [comment]
The necessary changes to support declaring one or more `rr:joinCondition`s have now been implemented in XRM.

Working example: Starting out from the example in the demo repository ...

> demo repo https://github.com/BenjaminHofstetter/xrm-parentTriplesMap

... and applying the following changes:

```diff
logical-source ParameterSource {
        referenceables
                id "@field-name"
                comment "@name"
+               stationId "../@number"
 }
```

```diff
map stationMapping from StationSource {
        properties
                ex.name from name ;
                ex.id from id ;
-               ex.hasParameter parent-map parameterMapping ;
+               ex.hasParameter parent-map parameterMapping join id with stationId ;
 }
```
![image](https://github.com/zazuko/xrm/assets/877232/f0a825d9-3f94-4dac-b99e-80c34260c2a9)

**Comment by ktk** [comment]
Great! I will have an XML to try it out as well, will talk to you next week :)

**Comment by mchlrch** [comment]
Resolved in xrm version 1.5.0
